src/Reducers/WishlistReducer.js

```diff
@@ -0,0 +1,9 @@
+const initWishlistState = []
+const wishlistReducer = (state, action) => {
```

Comment:
suggestion: giving state as the object will be a good practice, if we want to add any variable in the future, it would be easy. like initWishlistState = { wishlist: [ ] }

Reply:
okay will do